activeadmin.gemspec

```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'railties',            '>= 4.0', '< 5.1'
   s.add_dependency 'ransack',             '~> 1.3'
   s.add_dependency 'sass-rails'
+  s.add_dependency 'thor',                '0.19.1', '!=0.19.2', '!=0.19.3', '!=0.19.4' # http://stackoverflow.com/questions/41207432/
```

Comment:
Yeah, let's do that. I need to find time to propose the addition of lock files to the repo, but it definitely doesn't belong here.